packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -135,7 +136,7 @@ export const Control = ({
   );
 
   return (
-    <Container>
+    <Container $isExpanded={isExpanded}>
```

Comment:
In styled component, the dollar sign $ helps us to pass the parameter to the DOM through Material ui component. 
This is called the transient props, it is more like a defender to prevent passing everything to DOM: https://medium.com/@probablyup/introducing-transient-props-f35fd5203e0c
https://styled-components.com/docs/api#transient-props

Reply:
I didn't know that, that's awesome! 💯